README.md

```diff
@@ -1 +1,43 @@
-# gproject
\ No newline at end of file
+# Space-Travelers-Hub
+A web application that allow users to book rockets and join selected space missions, Using the SpaceX API.
+
+## Live Link
```

Comment:
- [ ] Please deploy the project online and include the live link in the readme.

Reply:
deployed